packages/pluggableWidgets/accordion-web/src/components/Accordion.tsx

```diff
@@ -21,11 +35,45 @@ export interface AccordionProps extends Pick<AccordionContainerProps, "class" |
 export function Accordion(props: AccordionProps): ReactElement | null {
     const reducer = useRef(getCollapsedAccordionGroupsReducer(props.singleExpandedGroup ? "single" : "multiple")); // the accordion group reducer function doesn't need to change during the lifetime of this component, since the singleExpandedGroup won't change.
 
-    const [collapsedAccordionGroups, collapsedAccordionGroupsDispatch] = useReducer(
+    const [accordionGroupCollapsedState, accordionGroupCollapsedStateDispatch] = useReducer(
         reducer.current,
-        props.groups.map(() => !props.previewMode && props.collapsible)
+        undefined,
+        () => {
+            const groupCollapsedStates = props.groups.map(
+                group => !props.previewMode && props.collapsible && !!group.initiallyCollapsed
```

Comment:
Nit: Could also make this `group.initiallyCollapsed === true` or keep it what you have. Just giving a suggestion if you think it is more readable than the other. I personally don't have a preference.

Reply:
I think your suggestion won't work, because the strict equal will not cast `group.initiallyCollapsed` to a boolean. You will need to use the `==` operator, which comes with some downsides. I will keep it as is.

Reply:
Fine to keep it as is. As I mentioned, it's more a nit. But to discuss whether it works:
From what I understand, you want to cast `group.initiallyCollapsed` to a boolean:
These are the results for `!!group.initiallyCollapsed`:
- !!true -> true
- !!false -> false
- !!undefined/null -> false

Similarly for `group.initiallyCollapsed === true`:
- true === true -> true
- false === true -> false
- undefined === true / undefined === true -> false

Reply:
Not sure if you saw this, so gonna ping you @IIsaku. You can resolve this whenever and merge aftwards.